src/kudu/consensus/consensus_queue.cc

```diff
@@ -492,6 +493,63 @@ Status PeerMessageQueue::AppendOperations(const vector<ReplicateRefPtr>& msgs,
   return Status::OK();
 }
 
+Status PeerMessageQueue::AppendOperation(const ReplicateMsgWrapper& msg_wrapper) {
```

Comment:
do we still need the ReplicateRefPtr versions of the same code?

Reply:
Not really, I kept it around for reference.